src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const takeOnlyMen
+    = century ? people.filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+      : people.filter(person => person.sex === 'm');
+
+  const getAges = takeOnlyMen.map(man => man.died - man.born);
```

Comment:
```suggestion
  const menAges = takeOnlyMen.map(man => man.died - man.born);
```
Please fix all cases. Don't start the name with a verb if it's not a function